components/automate-chef-io/content/docs/iam-actions.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "IAM Actions"
+description = "Reference Actions for Roles"
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "authorization"
+    weight = 30
++++
+
+Reference the chart on this page when creating a *Role* to know which action grants access to what page in the browser.
+
+*IAM Action* lists the associated action or actions required to access that page in the browser. 
+Use `*` in these actions to give broad permissions to perform all associated actions such as get, list, create, delete, etc.
+Specify the action to restrict user access to the specific action.
+
+
+|  Task           | Browser Tab     | IAM Action       | API endpoint  | URL       |
+| --------------- | --------------- | ---------------- | ------------- | --------- |
+| View Events | Dashboards | event:* | /event_feed | https://{{< example_fqdn "automate" >}}/dashboards/event-feed |
+| View Service Group Data | Applications | applications:*  | /applications/service-groups | https://{{< example_fqdn "automate" >}}/applications/service-groups |
+| View Client Runs | Infrastructure | infra:nodes:*   | /cfgmgmt/nodes | https://{{< example_fqdn "automate" >}}/infrastructure/client-runs |
+| Manage Chef Servers | Infrastructure | infra:servers:* | <endpoint> | https://{{< example_fqdn "automate" >}}/infrastructure/chef-servers |
+| View Reports | Compliance | compliance:reporting:*  | /compliance/reporting/reports | https://{{< example_fqdn "automate" >}}/compliance/reports/overview |
+| Review Scan Jobs | Compliance | [compliance:scannerJobs:* , infra:nodes:* , infra:nodeManagers:* , compliance:profiles:* ] | /compliance/scanner/jobs | https://{{< example_fqdn "automate" >}}/compliance/scan-jobs/jobs |
```

Comment:
why "review" for scan jobs ?  (vs view)
also wonder if we should do diff things for
view scan jobs list (which only requires compliance:scannerJobs:*)
vs 
edit/create scan jobs? (which requires all the same ones you have listed for review)

Reply:
Agree, "review" doesn't follow the patter. 
We use "view" to mean "show one" and "List" for "see more than one"

Reply:
I don't have an answer for "why review?" but I'm happy to change it.
Thanks @vj! Will modify for scan jobs accordingly.